lib/main.js

```diff
@@ -780,10 +824,10 @@ function StartQueue() {
   //
   // Get the renderer to use
   //
-  var renderer = (
-      (data.html || data.htmlNode || data.css) ? "CommonHTML" :
-          (data.svg || data.svgNode) ? "SVG" : "None"
-  );
+  var renderer = 'None';
+  if (data.html || data.htmlNode || data.css) renderer = "CommonHTML";
+  else if (data.svg || data.svgNode) renderer = "SVG"
+  else if (data.phtml || data.phtmlNode || data.pcss) renderer = "PreviewHTML"
```

Comment:
I think you need to swap lines 829 and 830, since you are checking for PHTML before SVG below (and the serenader routines assume the preference order is CHTML > PHTML > SVG).